tabbycat/api/urls.py

```diff
@@ -224,6 +224,14 @@
                         views.VenueCategoryViewSet.as_view(detail_methods),
                         name='api-venuecategory-detail'),
                 ])),
+                path('/actionlog', include([
+                    path('',
+                        views.ActionLogViewSet.as_view(list_methods),
```

Comment:
```suggestion
                        views.ActionLogViewSet.as_view({'get': 'list'}),
```
By only keeping the `get` verb, the API views become read-only.